readme.md

```diff
@@ -507,6 +507,7 @@ console.log(fileType);
 - [`vcf`](https://en.wikipedia.org/wiki/VCard) - vCard
 - [`voc`](https://wiki.multimedia.cx/index.php/Creative_Voice) - Creative Voice File
 - [`vsdx`](https://en.wikipedia.org/wiki/Microsoft_Visio) - Microsoft Visio File
+- [`vtt`](https://www.w3.org/TR/webvtt1/) - WebVTT File (for video captions)
```

Comment:
It looks like the convention is to link to wikipedia 
```suggestion
- [`vtt`](https://en.wikipedia.org/wiki/WebVTT) - Web Video Text Tracks, a W3C standard file for subtitles or captures
```

Reply:
Yup, I noticed that too. The only reason I didn't do that is because the Wikipedia article doesn't list the magic numbers for this file format, so I thought it would be more helpful to link to the w3 spec.

Reply:
It also the formal standard, so I fully get you. But it is not lost, it remains documented as part of this PR.

I think the function of this link, in addition as a reference, the links servers to give an context to ensure there can be no ambiguity which file type we are talking about.